docs/configuration/outbox-event-router.asciidoc

```diff
@@ -137,7 +139,7 @@ This result was achieved with the link:#default-table-columns[default configurat
 |The default regex to use within the RegexRouter, the default capture will allow to replace the routed field into a new topic name defined in `route.topic.replacement`
 
 |`route.topic.replacement`
-|`outbox.event.${routedByValue}`
+|`outbox.event{zwsp}.${routedByValue}`
```

Comment:
What's `zwsp`?

Reply:
@renatomefi zero whitespace - allows to break line in this place for formatting